src/Socket.cpp

```diff
@@ -36,7 +36,10 @@ static CString ZNC_DefaultCipher() {
     return "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:"
            "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:"
            "ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:"
-           "DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384";
+           "DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:"
```

Comment:
So this is not a copypaste from mozilla page anymore?

Reply:
Chacha20 is a good one, under certain aspects, better then AES too.
It has been implemented for ECDHE version too.
I checked a bit, but I find out no good reasons to exclude it.
It is still a "copy/paste", with just the add of Chacha20 for the DHE.